Doc/library/sqlite3.rst

```diff
@@ -1316,31 +1317,13 @@ Connection objects
 
    .. attribute:: row_factory
 
-      A callable that accepts two arguments,
-      a :class:`Cursor` object and the raw row results as a :class:`tuple`,
-      and returns a custom object representing an SQLite row.
-
-      Example:
-
-      .. doctest::
-
-         >>> def dict_factory(cursor, row):
-         ...     col_names = [col[0] for col in cursor.description]
-         ...     return {key: value for key, value in zip(col_names, row)}
-         >>> con = sqlite3.connect(":memory:")
-         >>> con.row_factory = dict_factory
-         >>> for row in con.execute("SELECT 1 AS a, 2 AS b"):
-         ...     print(row)
-         {'a': 1, 'b': 2}
-
-      If returning a tuple doesn't suffice and you want name-based access to
-      columns, you should consider setting :attr:`row_factory` to the
-      highly optimized :class:`sqlite3.Row` type. :class:`Row` provides both
-      index-based and case-insensitive name-based access to columns with almost no
-      memory overhead. It will probably be better than your own custom
-      dictionary-based approach or even a db_row based solution.
+      Define the default :attr:`Cursor.row_factory`
+      for cursors created off of this connection.
+      Assigning to this attribute does not affect the :attr:`!row_factory`
+      of open cursors belonging to this connection.
```

Comment:
```suggestion
      of existing cursors belonging to this connection, only new ones.
```

* Explicitly contrast with what it _does_ affect
* Tweak wording slightly to be more precise (as I understand it, whether the cursor is affected depends on whether the cursor has already been created, not whether `close()` has been called on it as this would imply) and avoid unnecessary confusion (readers might not necessarily be familiar with open/`close()` semantics, whereas simple existence is much more basic and obvious)

Reply:
+1

> [...] as I understand it, whether the cursor is affected depends on whether the cursor has already been created, not whether close() has been called on it as this would imply [...]

This is correct.

Reply:
However, I find the _"only new ones"_ a little bit weird. I'll try to come up with an alternative.